internal/apis/certmanager/validation/issuer.go

```diff
@@ -347,6 +347,33 @@ func ValidateVenafiTPP(tpp *certmanager.VenafiTPP, fldPath *field.Path) (el fiel
 
 	// TODO: validate CABundle using validateCABundleNotEmpty
 
+	// Validate only one of CABundle, CABundleSecretRef and CABundleConfigMapRef is passed
+	el = append(el, validateVenafiTPPCABundleUnique(tpp, fldPath)...)
+
+	return el
+}
+
+func validateVenafiTPPCABundleUnique(tpp *certmanager.VenafiTPP, fldPath *field.Path) (el field.ErrorList) {
+	numCAs := 0
+	if len(tpp.CABundle) > 0 {
+		numCAs++
+		if numCAs > 1 {
+			el = append(el, field.Invalid(fldPath.Child("caBundle"), "<snip>", "may not specify more than one of CABundle/CABundleSecretRef/CABundleConfigMapRef as CA Bundle"))
```

Comment:
Could you move this check after the if statements and have a single invalid field when multiple values are provided? (see https://github.com/cert-manager/cert-manager/pull/6454/files#diff-6f1643a18e06442b93897a9b9f7bb15a1dc3a0f40b93f357110985eee7d53cd3R325)